ISISNeutronMuon/InstrumentScripts
# Ticket5404

### Instrument(s)

Any instruments which use scans library, reflectometry

### Story/Acceptance criteria

As a user of these scripts I want to be able to be able to keep using them after IBEX has transitioned to python 3.

Acceptance criteria:
- [ ] Code is python 2 and 3 compatible
- [ ] Issues highlighted in https://github.com/ISISComputingGroup/IBEX/issues/5271#issuecomment-593911766 are addressed
- [ ] Tests are implemented for any change

### Description of work

futurize was ran on all scripts in this repo. Any suggested changes were either implemented with a test, or ignored as inappropriate. In these cases, a manual test was performed to make sure the functionality is the same in python 2 and 3 without modification.

### Issue/Ticket Reference

https://github.com/ISISComputingGroup/IBEX/issues/5404

### Tests

Unit tests where possible, otherwise a manual test was done to make sure the code didn't need changing. For example, in the larmor script I made sure that the script behaved the same with and without adding `list()`.

### To test

* Run all unit tests, check they pass
* Check that the points in https://github.com/ISISComputingGroup/IBEX/issues/5271#issuecomment-593911766 have been addressed.

---

#### Code Review


### Final Steps
